packages/esm-patient-labs-app/src/test-results/panel-timeline/timeline.scss

```diff
@@ -75,6 +74,9 @@
   background-color: $color-gray-30;
   grid-template: auto auto / 9em auto;
   border-collapse: collapse;
+  
+  width: fit-content;
```

Comment:
Looks good to me. Thanks @suubi-joshua   , cc @denniskigen

Reply:
Shouldn't all panels consistently take up the entire available space without fitting to the individual content sizes?
More so like:
<img width="1440" alt="Screenshot 2024-07-19 at 7 03 00 PM" src="https://github.com/user-attachments/assets/4c12eb5b-e74d-4fcd-b02b-1db7633bc709">

Reply:
Yes this looks more neat. @NethmiRodrigo